### PR TITLE
Updade Readme.md

### PR DESCRIPTION
**Pull request by Dmitry-White**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,3 @@
 # Basic project setup
-## Hello World
\ No newline at end of file
+## Hello World
```
Comment (cqxg) [inline README.md:2]: There's a strange new "Hello World" line here.

Comment (Dmitry-White) [reply README.md:2]: It's ok

Comment (cqxg) [reply README.md:2]: Well then you're good to go!

Comment (Dmitry-White) [reply README.md:2]: Thank! Starting merge

**Pull request merged**